### PR TITLE
New chat

### PR DESCRIPTION
**Pull request by PatreevIgor**

### CODE REVIEW

File: app/channels/application_cable/connection.rb [hunk 1]
```diff
@@ -2,5 +2,26 @@
...
+
+    def connect
+      self.current_user = find_verified_user
+      self.current_chat = find_current_chat
+      logger.add_tags 'ActionCable', current_user.email
+    end
+
+    protected
+    def find_verified_user
+      if (current_user = User.find_by_id cookies.signed['user.id'])
```
Comment (MikeAndrianov) [inline app/channels/application_cable/connection.rb]: `User.find_by_id cookies.signed['user.id'] || reject_unauthorized_connection`

File: app/channels/application_cable/connection.rb [hunk 2]
```diff
@@ -2,5 +2,26 @@
...
+        current_user
+      else
+        reject_unauthorized_connection
+      end
+    end
+
+    def find_current_chat
+      Chat.all.find_by board_id: cookies[:board_id]
```
Comment (MikeAndrianov) [inline app/channels/application_cable/connection.rb]: `Chat.find_by ...`

File: app/controllers/dashboard_controller.rb [hunk 3]
```diff
@@ -32,8 +34,12 @@ def create
     end
   end
 
+  def create_message
+    Message.create(text: data['message'], user_id: current_user.id, chat_id: Chat.find_by_id(session[board_id]).id)
```
Comment (MikeAndrianov) [inline app/controllers/dashboard_controller.rb]: `find_by(id: id)`

File: app/views/dashboard/show.html.haml [hunk 4]
```diff
@@ -1,7 +1,10 @@
 .container-fluid
   =render 'shared/menu_bar'
   %h1 Your lists
-  .row
-    -@lists.each do |list|
-      =render 'shared/list', list: list, notes: list.notes
-    =render 'shared/list_form'
+  #row_for_list
+    .row
+      -@lists.each do |list|
```
Comment (MikeAndrianov) [inline app/views/dashboard/show.html.haml]: render collection

File: app/views/lists/index.html.haml [hunk 5]
```diff
@@ -1,6 +1,8 @@
 .container-fluid
   %h1 Your lists
-  .row
-    - @lists.each do |list|
-      = render 'shared/list', list: list, notes: list.notes
-    = render 'shared/list_form'
+  #row_for_chat
+    .row
+      - @lists.each do |list|
```
Comment (MikeAndrianov) [inline app/views/lists/index.html.haml]: render collection

File: app/views/dashboard/show.html.haml (hunk 4 above)
Comment (PatreevIgor) [inline app/views/dashboard/show.html.haml]: this I did and in the test branch these changes have already been made. But I just found out that it is possible, thank you. When you merge I will solve this conflict

**Pull request merged**